ab3d2_source/hires.s

```diff
@@ -217,6 +217,9 @@ xwobxoff:					dc.w	0
 xwobzoff:					dc.w	0
 CollId:						dc.w	0
 
+LookMin:	dc.w 0
```

Comment:
Would you mind giving these a name something like View_LookMin_w ?

Reply:
suggested renaming done.